src/components/CompareResults/RunValues.tsx

```diff
@@ -63,6 +64,9 @@ function RunValues(props: RunValuesProps) {
             </div>
           ))}
         </div>
+        <div>
+          <b>Mean</b>: {avg} <b>Median</b>: {median}
```

Comment:
nits: add units, also add a comma after the mean value to better visually separate the 2 values.